hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/WriteFileThroughputBenchmark.java

```diff
@@ -0,0 +1,208 @@
+package org.apache.hadoop.ozone.freon;
+
+import com.codahale.metrics.Timer;
+import org.apache.hadoop.fs.FSDataOutputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdds.cli.HddsVersionProvider;
+import org.apache.hadoop.hdds.conf.OzoneConfiguration;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+import picocli.CommandLine.Option;
+import org.slf4j.Logger;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.net.URI;
+import java.util.concurrent.Callable;
+
+/**
+ * File Generator for populating the cluster with random files.
+ */
+@CommandLine.Command(name = "wtb",
+    aliases = "write-throughput-benchmark",
+    description = "Benchmark for creating a file",
+    versionProvider = HddsVersionProvider.class,
+    mixinStandardHelpOptions = true,
+    showDefaultValues = true)
+public class WriteFileThroughputBenchmark extends BaseFreonGenerator
+    implements Callable<Void> {
+
+  @Option(names = {"-o"},
```

Comment:
As this option is for rootPath can we change '-o' to '-p' and also we can add a second parameter like '--path'
like names = {"-p", "--path"}

Reply:
Thanks for the suggestion, I will do that 👍